src/main/java/seedu/address/ui/StatsWindow.java

```diff
@@ -0,0 +1,104 @@
+package seedu.address.ui;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.logging.Logger;
+
+import javafx.scene.Scene;
+import javafx.scene.chart.BarChart;
+import javafx.scene.chart.CategoryAxis;
+import javafx.scene.chart.NumberAxis;
+import javafx.scene.chart.XYChart;
+import javafx.scene.paint.Color;
+import javafx.stage.Stage;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.Model;
+import seedu.address.model.flashcard.RatingContainsKeywordPredicate;
+
+//@@author LeonardTay748
+/**
+ * Displays Statistics as Bar Chart
+ */
+public class StatsWindow extends UiPart<Stage> {
+
+    private static final Logger logger = LogsCenter.getLogger(StatsWindow.class);
+    private static final String FXML = "StatsWindow.fxml";
+    private static final Stage stage = new Stage();
+
+    private static final String good = "Good";
+    private static final String hard = "Hard";
+    private static final String easy = "Easy";
+
+
+    /**
+     * Creates a new StatsWindow.
+     */
+    public StatsWindow() {
+        super(FXML, stage);
+    }
+
+    /**
+     * Shows the stats window.
+     */
+    public void show(Model model) {
+        logger.fine("Showing stats page about the application.");
+
+        requireNonNull(model);
```

Comment:
You might want this to be the first line in the method - if the model turns out to be null, you probably shouldn't log that you are showing the stats page.